web-app/js/portal/details/NcWmsPanel.js

```diff
@@ -323,7 +329,8 @@ Portal.details.NcWmsPanel = Ext.extend(Ext.Container, {
         if (this.geoNetworkRecord) {
             this._addDateTimeFilterToLayer();
             this.geoNetworkRecord.updateNcwmsParams(dateRangeStart, dateRangeEnd, geometry);
-            Ext.MsgBus.publish(PORTAL_EVENTS.DATA_COLLECTION_MODIFIED);
+            // todo cant have this for accordion
+            //Ext.MsgBus.publish(PORTAL_EVENTS.DATA_COLLECTION_MODIFIED);
```

Comment:
Perhaps removing this comment would be better.  It won't mean much to anyone after this change is merged.